push/sender/src/main/java/org/jboss/aerogear/unifiedpush/message/sender/APNsPushNotificationSender.java

```diff
@@ -209,4 +205,43 @@ public void messageSendFailed(ApnsNotification message, Throwable e) {
         // null if, why ever, there was no cert/passphrase
         return null;
     }
+
+    /**
+     * Configure the Gateway to the Apns servers.
+     * Default gateway and port can be override with respectively :
+     *  - aerogear.apns.push.host
+     *  - aerogear.apns.push.port
+     *
+     * Feedback gateway and port can be override with  respectively :
+     *  - aerogear.apns.feedback.host
+     *  - aerogear.apns.feedback.port
+     * @param iOSVariant
+     * @param builder
+     */
+    private void configureDestinations(iOSVariant iOSVariant, ApnsServiceBuilder builder) {
+        // pick the destination:
+        if (iOSVariant.isProduction()) {
+            builder.withProductionDestination();
+        } else {
+            builder.withSandboxDestination();
+        }
+
+        //Is the gateway host&port provided by a system property ?
+        if(System.getProperty("aerogear.apns.push.host")!= null){
+            int port = Utilities.SANDBOX_GATEWAY_PORT;
+            if(System.getProperty("aerogear.apns.push.port")!= null) {
```

Comment:
Might be a good thing to take these strings and put them as constants at the top of this file. Firstly it would be easier to change in the future (if needed) and secondly it will be easily seen on top of the file. But I'll leave that decision to you :).